### DISCUSSION TITLE
Help trying to add support for additional language

### DISCUSSION THREAD

**Discussion by hershkoy** [microsoft/Recognizers-Text#2697] [General]
I am trying to add support for Hebrew, following the instructions given in the [Contributing to LUIS with Microsoft/Recognizers-Text](https://blog.botframework.com/2018/01/24/contributing-luis-microsoft-recognizers-text-part-1/).

I forked the repo to my account. Currently working on the Numbers module. 
You can see my changes [in the commit](https://github.com/hershkoy/Recognizers-Text/commit/fe9939da8f8b3be0e40685fefb681c6d20340fc2)

I run `npm run build` to compile the project, and it generated the hebrew compiled folder:

![image](https://user-images.githubusercontent.com/13259380/130345408-f08b922f-1591-4e23-86bb-897e9c103755.png)

However it doesn't work. I tried running the test, but it just 'skips' the Hebrew one. 

So I tried running specifically the Hebrew test using, and I am getting error that there are no tests:

```
npm test test/runner.js --match='*Hebrew*'

  × Couldn’t find any matching tests
```

I tried changing JavaScript\samples\simple-console\index.js and switched the defaultCulture:
```
const defaultCulture = Recognizers.Culture.Hebrew;
```
But it is not recognizing my Hebrew numbers.

What am I missing?

**Comment by tellarin** [comment]
Hi @hershkoy, thanks for contributing to the project.

One item that is unclear in that post is that you need to register the new language in the main recognizers class for each type and also with the test running code. Below is a list of related files:
- ./test/cultures.js
- ./recognizers-text/src/culture.ts
- ./recognizers-number/src/number/numberRecognizer.ts

You also need to further edit ./recognizers-number/src/recognizers-text-number.ts 
To ```export { HebrewNumeric } from "./resources/hebrewNumeric";```.

Feel free to open a draft PR and we can take a look and add comments in the missing pieces.

**Comment by tellarin** [comment]
@hershkoy did this help unblock you?

**Reply by hershkoy** [reply]
Sorry, I had a busy week
I added some more places where I missed adding the Hebrew definitions, but it still doesn't work.
I will try to update soon.
About the pull request: Should I do it in a separate branch or something?

**Reply by tellarin** [reply]
In your fork of the repo, create a new branch, then make a PR from there and mark it as Draft. I'll try to take a closer look ASAP.

**Reply by hershkoy** [reply]
done:
PR: https://github.com/microsoft/Recognizers-Text/pull/2699
